### PR TITLE
Add Singapore

### PR DESCRIPTION
**Pull request by pdxlocations**

### CODE REVIEW

File: docs/blocks/_lora-regions.mdx [hunk 1]
```diff
@@ -17,6 +17,7 @@
 |  `UA_868`   |     Ukraine 868MHz      |
 |  `MY_433`   |     Malaysia 433MHz     |
 |  `MY_919`   |     Malaysia 919MHz     |
+|  `SG_923`   |     Singapore 923MHz     |
```
Comment (tekstrand) [inline docs/blocks/_lora-regions.mdx:20]: Extra space 🍺

File: docs/hardware/devices/nano/index.mdx [hunk 2]
```diff
@@ -36,7 +36,7 @@ The Nano G2 Ultra and Nano G1 Explorer have exactly the same Lora front-end circ
 - **LoRa Transceiver:**
   - Semtech SX1262
 - **Frequency Options:**
-  - _US (902.0Mhz to 928.0Mhz), EU_868 (869.4Mhz to 869.65Mhz), JP (920.8Mhz to 927.8Mhz), ANZ (915.0Mhz to 928.0Mhz), RU (868.7Mhz to 869.2Mhz), KR (920.0Mhz to 923.0Mhz), TW (920.0Mhz to 925.0Mhz), IN (865.0Mhz to 867.0Mhz), NZ_865 (864.0Mhz to 868.0Mhz), TH (920.0Mhz to 925.0Mhz), UA_868 (868.0Mhz to 868.6Mhz)._
+  - _US (902.0Mhz to 928.0Mhz), EU_868 (869.4Mhz to 869.65Mhz), JP (920.8Mhz to 927.8Mhz), ANZ (915.0Mhz to 928.0Mhz), RU (868.7Mhz to 869.2Mhz), KR (920.0Mhz to 923.0Mhz), TW (920.0Mhz to 925.0Mhz), IN (865.0Mhz to 867.0Mhz), NZ_865 (864.0Mhz to 868.0Mhz), TH (920.0Mhz to 925.0Mhz), UA_868 (868.0Mhz to 868.6Mhz), MY_919 (919.0Mhz to 924.0Mhz), SG_923 (917.0Mhz to 925.0Mhz)._
```
Comment (tekstrand) [inline docs/hardware/devices/nano/index.mdx:39]: What is `MY`? Was that change intentional?

**Review by tekstrand — APPROVED** [review]
One question and one suggestion but nothing blocking 👍

File: docs/hardware/devices/nano/index.mdx (hunk 2 above)
Comment (pdxlocations) [reply docs/hardware/devices/nano/index.mdx:39]: Malaysia was added a bit ago but I missed this reference so picked it off while I was in the neighborhood.

File: docs/blocks/_lora-regions.mdx (hunk 1 above)
Comment (pdxlocations) [reply docs/blocks/_lora-regions.mdx]: how dare i
